src/olympia/constants/base.py

```diff
@@ -212,16 +212,16 @@
     'utm_source': 'addons.mozilla.org',
 }
 VALID_CONTRIBUTION_DOMAINS = (
-    'buymeacoffee.com',
+    'www.buymeacoffee.com',
     'donate.mozilla.org',
     'flattr.com',
     'github.com',
     'ko-fi.com',
     'liberapay.com',
-    'micropayment.de',
+    'www.micropayment.de',
     'opencollective.com',
-    'patreon.com',
-    'paypal.com',
+    'www.patreon.com',
+    'www.paypal.com',
     'paypal.me',
```

Comment:
In the migration, you're converting `paypal.me` to `www.paypal.com`, so should we be keeping `paypal.me` here? And if so, why are we converting it in the migration?

Reply:
We shouldn't convert it: paypal started redirecting `https://paypal.me/foo` to `https://www.paypal.com/paypalme/foo` at some point, but I don't think we should be trying to do that conversion ourselves. And just replacing `https://paypal.me/foo` with `https://paypal.com/foo` is wrong.

Reply:
whoops (it's a mistake in the dict in the migration)

Reply:
https://github.com/mozilla/addons-server/pull/19006/commits/082e8b7ee99b05f7e54b8c18c338bd6a6a470f83